src/webhooks.ts

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+  MarqetaError,
+} from './'
+
+export interface WebHook {
+  name: string;
+  token?: string;
+  active?: boolean;
+  config: {
+    url?: string;
+    secret?: string;
+    basicAuthUsername: string;
+    basicAuthPassword: string;
+    customHeader: any;
+    useMtls: boolean;
+  },
+  events: string[];
+}
+
+export interface BusinessList {
+  count: bigint;
+  startIndex: bigint;
+  endIndex: bigint;
+  isMore: boolean;
+  data?: WebHook[];
+}
```

Comment:
Is the `BusinessList` not defined in the Business module? I would not imagine it'd be defined in the webhook module... or am I missing something?

I think you might find this better called `WebhookList`... I think...